### PR TITLE
P: podleze-piekielko.pl (scrolling blocked by Cookie List)

### PR DESCRIPTION
**Pull request by peace2000**
https://podleze-piekielko.pl/

There needs to be whitelisting for other blockers: https://github.com/easylist/easylist/issues/8291

Edit, also added a rule for Ubo to counter this whitelisting rule.

Edit2: removed all dom and style rules and added scriptlets for both ABP and Ubo.

### CODE REVIEW

**Comment by krystian3w** [comment]
Events"load" tested same good to "`esbo.fi,espoo.fi`" (I try exclude broken links/buttons/comments then words if pop-up are higieny by CSS).

**Comment by peace2000** [comment]
Oh... Yeah scriplet removes arrows:

kuva

kuva

So I'll remove it and stick with style rules instead...

**Comment by ryanbr** [comment]
@peace2000   relook at this, also `podleze-piekielko.pl###cookieman-modal, important` is invalid, important cant be used with cosmetics

**Comment by krystian3w** [comment]
Valid possible create <important> tag by Javascript.

**Comment by peace2000** [comment]
@ryanbr Ubo supports that with cosmetics.

**Comment by krystian3w** [comment]
This only trie found no exist html element, ABP also support combine many selectors by "`, `" long time ago (for better compatiblity don't remove space due compress file).

---

other way is duplicate or use `[attr]`:

`###cookieman-modal#cookieman-modal`
`###cookieman-modal[id]`

rather use name of element like "div":

`##div#cookieman-modal`

also maybe safer is:

`apor, cookieman` if buttons/sliders works fine.

**Comment by peace2000** [comment]
I don't see any issues with _podleze-piekielko.pl##+js(aopr, cookieman)_ so I think I'll add it then.

I'll also add an ABP equivalent.

**Comment by peace2000** [comment]
Ok now it's fine and ready to go @ryanbr

**Comment by krystian3w** [comment]
I suppose no possible use spinnets other place than anti-CV (ABP Filters) + manually added filters on list "my filters" to 2023 or longer (gitlab team a little scary global usage spinnets by any project - I suppose try only extend support recommend lists for 1 to more in future).

**Review by krystian3w — CHANGES_REQUESTED** [review]

File: easylist_cookie/easylist_cookie_specific_ABP.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 ! ABP Sepecific fixes
 minhaconexao.com.br#?#section[id][class]:-abp-has(:scope > div[class] > button[class][onclick^="onClickCookiesBannerWeb"])
+! Scripts
+podleze-piekielko.pl#$#abort-on-property-read cookieman
```
Comment (krystian3w) [inline easylist_cookie/easylist_cookie_specific_ABP.txt:4]:
```suggestion
```

may no work before 2022/2023 year for ABP users with normal subscription.

**Review by krystian3w — CHANGES_REQUESTED** [review]

File: easylist_cookie/easylist_cookie_specific_ABP.txt (hunk 1 above)
Comment (peace2000) [reply easylist_cookie/easylist_cookie_specific_ABP.txt:4]: I don't understand? If I load ABP and apply that filter, it works fine.

**Comment by peace2000** [comment]
@krystian3w Imo those fears sound pretty far-fetched. I'll let @ryanbr decide whether that entry for ABP users will be accepted or not. I don't see any issues with it, as currently any ABP user can use that filter anyway and it works just fine. If it works, I don't understand why iit would be "forbidden". Easylist Cookie has ABP section anyway, which is dedicated for ABP specific solutions.

File: easylist_cookie/easylist_cookie_specific_ABP.txt (hunk 1 above)
Comment (krystian3w) [reply easylist_cookie/easylist_cookie_specific_ABP.txt:4]: Try load from commit link and test page, not form "my custom added list".

**Comment by krystian3w** [comment]
But this project is allowed to load sinppets from online files: 

https://gitlab.com/eyeo/adblockplus/adblockpluscore/-/tree

commit from addon resource please.

**Comment by spirillen** [comment]
@peace2000 Rebase...

**Comment by krystian3w** [comment]
Rebase no disable waiting then spinnet run without manually copy from file to "my filters" in ABP settings.

**Comment by peace2000** [comment]
Rebasing didn't went properly. Nah, I'll make a new pull req. Maybe better make two different in case snippet usage allowance is uncertain in ABP...

**Comment by krystian3w** [comment]
`warta.pl` has it.

**Pull request closed without merging**